pkg/packagekit/mock-updates.es6

```diff
@@ -0,0 +1,60 @@
+/*jshint esversion: 6 */
+
+// Mock "available updates" entries for interactively testing layout changes with large updates
+export function injectMockUpdates(updates) {
+    // two security updates
+    updates["security-one;2.3-4"] = {
+        name: "security-one",
+        version: "2.3-4",
+        bug_urls: [],
+        cve_urls: ["https://cve.example.com?name=CVE-2017-0815"],
+        security: true,
+        description: "OMGkittensdie",
```

Comment:
personal gripe: could we use an example that doesn't involve death? `WorldOfHurt`, `AcidBurn` et cetera